__init__.py

```diff
@@ -119,6 +119,8 @@ def gbp():
     'WDR': ('WDR', 'https://www1.wdr.de/mediathek/audio/'
                    'wdr-aktuell-news/wdr-aktuell-152.podcast',
             image_path('WDR')),
+    'hr-iNFO': ('hr-iNFO', 'https://podcast.hr-online.de/der_tag_in_hessen/podcast.xml',
```

Comment:
Maybe the key could be shortened to `HRI`?
From memory this is only really used to match the Skill setting to the station. The voice commands are based on the first element of the tuple which would remain 'hr-info'

If there's other longer names that people might use to reference the station you can also add this into [`dialog/lang/alt.feed.name.value`](https://github.com/MycroftAI/skill-npr-news/blob/20.08/dialog/en-us/alt.feed.name.value)

Reply:
I've updated PR based on your suggestions.